src/test/java/ua/com/juja/microservices/keepers/slackbot/model/SlackParsedCommandTest.java

```diff
@@ -91,22 +94,19 @@ public void getFromUser() {
     @Test
     public void getUserCount() {
         //given
-        String text1 = "text @slack text";
-        String text2 = "@slack text @slack text @slack";
+        String text = "text @slack text";
```

Comment:
the text contains only one slackName, but in the test
assertEquals(3, slackParsedCommand.getUserCountInText());